include/gc/Dialect/OneDNNGraph/OneDNNGraphOps.td

```diff
@@ -0,0 +1,82 @@
+//===- OneDNNGraphOps.td - OneDNN input dialect ops --------*- tablegen -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef ONEDNNGRAPH_OPS
+#define ONEDNNGRAPH_OPS
+
+include "mlir/Interfaces/InferTypeOpInterface.td"
+include "mlir/Interfaces/SideEffectInterfaces.td"
+include "mlir/Interfaces/DestinationStyleOpInterface.td"
+include "mlir/IR/AttrTypeBase.td"
+include "mlir/IR/OpBase.td"
+include "OneDNNGraphDialect.td"
+include "OneDNNGraphTypes.td"
+
+//===----------------------------------------------------------------------===//
+// Base OneDNNGraph operation definition.
+//===----------------------------------------------------------------------===//
+
+class OneDNNGraph_Op<string mnemonic, list<Trait> traits = []> :
+        Op<OneDNNGraphDialect, mnemonic, traits>;
+
+class OneDNNGraph_ElemwiseBinaryOp<string mnemonic, list<Trait> traits = []> :
+    OneDNNGraph_Op<mnemonic, traits # [SameOperandsAndResultElementType, InferTensorType]> {
+  let arguments = (ins OneDNNGraph_LogicalTensor:$input_0, 
+                       OneDNNGraph_LogicalTensor:$input_1);
+  let results = (outs OneDNNGraph_LogicalTensor:$result);
+
+  let assemblyFormat =
+      "operands attr-dict `:` functional-type(operands, results)";
```

Comment:
Do we really need it?

Reply:
Yeah, I think making assembly format explicit would be better.